packages/orbit-components/src/Accordion/AccordionSection/components/SectionHeader.tsx

```diff
@@ -1,42 +1,8 @@
 import React from "react";
-import styled, { css } from "styled-components";
+import cx from "clsx";
 
 import Button from "../../../Button";
 import type * as Common from "../../../common/types";
-import defaultTheme from "../../../defaultTheme";
-
-const StyledWrapper = styled.div<{
-  expanded: boolean;
-  "data-test"?: string | boolean;
-}>`
-  ${({ theme, expanded }) => css`
-    display: flex;
-    padding: ${theme.orbit.spaceLarge};
-    background-color: ${theme.orbit.paletteWhite};
-    align-items: center;
-    min-height: ${expanded ? "19px" : "44px"};
-    transition: max-height ${theme.orbit.durationFast} ease-in-out;
```

Comment:
Turned out this transition was also useless. In the end all the transitions are handled only by the `<Slide` inside the `AccordionSection`.